src/index.md

```diff
@@ -333,6 +333,115 @@ For the purposes of brevity this document will use the term "bug" to refer to bo
 
 _Congratulations!_ Your changeset will now make its way towards a promoted build. When the changeset becomes part of a promoted build, the bug's "Resolved in Build" will have a value of \"b\[1-9\]\[0-9\]&ast;\" to indicate the build number.
 
+# Testing Changes
+
+In addition to your own Java applications, OpenJDK have support for two test frameworks, JTReg and GTest. JTReg is a Java regression test framework that is used for most of the tests that are included in the OpenJDK source repository. The Google Test (GTest) framework is intended for unit testing of the C++ native code.
+
```

Comment:
There's lots of C++ code in the JDK client - some of it shared, some of it windows specific. None of that is set up or likely appropriate for gtest to test it so if you are thinking of hotspot C++ code may be you should say that ..

Reply:
Although you updated Gtest's introduction paragraph to reflect that as of now gtest is for hotspot only, I think it needs to be said here as well.

> The Google Test (GTest) framework is intended for unit testing of the C++ native code.

this statement can also create an inadequate impression that you can't/shouldn't test native code w/ jtreg, this isn't the case as one can a) test native code thru java API; b) use native test code in jtreg tests. 

btw, point b) might deserve a sentence or two in JTreg section of this document.

Reply:
Please provide a sentence or two about b). Or maybe add that as a separate change so that you get proper credit for it and can become a Committer soon :-)